ibm/service/eventnotification/resource_ibm_en_smtp_configuration.go

```diff
@@ -0,0 +1,414 @@
+// Copyright IBM Corp. 2024 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package eventnotification
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/validate"
+	en "github.com/IBM/event-notifications-go-admin-sdk/eventnotificationsv1"
+	"github.com/IBM/go-sdk-core/v5/core"
+)
+
+func ResourceIBMEnSMTPConfiguration() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMEnSMTPConfigurationCreate,
+		ReadContext:   resourceIBMEnSMTPConfigurationRead,
+		UpdateContext: resourceIBMEnSMTPConfigurationUpdate,
+		DeleteContext: resourceIBMEnSMTPConfigurationDelete,
+		Importer:      &schema.ResourceImporter{},
+		Timeouts: &schema.ResourceTimeout{
```

Comment:
Why were this timeouts added I don't see any wait functions to use these timeouts